server/src/Application/Services/Quiz/IQuizService.cs

```diff
@@ -0,0 +1,14 @@
+﻿using StudyZen.Application.Dtos;
+
+namespace StudyZen.Application.Services;
+
+public interface IQuizService
+{
+    Task<QuizDto> CreateQuiz(CreateQuizDto dto);
+    Task<QuizDto> GetQuiz(int quizId);
+    Task<IReadOnlyCollection<QuizDto>> GetAllQuizzes();
+    Task UpdateQuiz(int quizId, UpdateQuizDto dto);
+    Task DeleteQuiz(int quizId);
+    Task<QuizQuestionDto> AddQuestionToQuiz(int quizId, CreateQuizQuestionDto dto);
```

Comment:
Maybe add a delete question by id function too?

Reply:
Yeah, but maybe in the future when it's used